Packs/Rundeck/Integrations/Rundeck/Rundeck.py

```diff
@@ -0,0 +1,1113 @@
+        if not file_path:
+            raise DemistoException(
+                f"Could not find file path to the next entry id: {entry_id}. \n"
+                f"Please provide another one."
+            )
+        else:
+            file_name = ntpath.basename(file_path)
+
+        request_params.update(self.params)
+        del self._headers['Content-Type']
+        with open(file_path, "rb") as file:
+            self._headers.update({'Content-Disposition': f'form-data; name="file"; filename="{file_name}"'})
+            return self._http_request(
+                method="POST",
+                files={'scriptFile': file},
+                url_suffix=f'/project/{project_name_to_pass}/run/script',
+                params=request_params
+            )
+
+
+''' HELPER FUNCTIONS '''
+
+
+def filter_results(results: Union[list, dict], fields_to_remove: list, remove_signs) -> Union[list, dict]:
+    new_results = []
+    if isinstance(results, dict):
+        demisto.info('got results as dictionary')
+        new_record = {}
+        demisto.info('start looping over results')
+        for key, value in results.items():
+            if key not in fields_to_remove:
+                demisto.debug(f'add this key: "{key}" to filtered results')
+                if isinstance(value, dict):
+                    demisto.debug(f'found {value} is a dict, calling this function again')
+                    value = filter_results(value, fields_to_remove, remove_signs)
+                demisto.info('searching not allowed signs to remove')
+                for sign in remove_signs:
+                    if sign in key:
+                        demisto.debug(f'found "{sign}" in the next key: "{key}". remove it.')
+                        new_record[key.replace(sign, '')] = value
+                        demisto.debug('finish remove it')
+                    else:
+                        demisto.debug(f'not allowed signs were not found. add the next key to filter results: {key}')
+                        new_record[key] = value
+                demisto.info('finish remove not allowed signs in results keys')
+        demisto.info('finish looping over results')
+        return new_record
+    else:
+        demisto.info('got results as list')
+        for record in results:
+            new_record = {}
+            for key, value in record.items():
+                if key not in fields_to_remove:
+                    if isinstance(value, dict):
+                        value = filter_results(value, fields_to_remove, remove_signs)
+                    for sign in remove_signs:
+                        if sign in key:
+                            new_record[key.replace(sign, '')] = value
+                        else:
+                            new_record[key] = value
+            new_results.append(new_record)
+    return new_results
+
+
+def attribute_pairs_to_dict(attrs_str: Optional[str], delim_char: str = ","):
```

Comment:
please remove logs from this function. this is an internal object manipulation function, no need for logging.
as long as its not errors. then we will want to add the input arguments in the error.